README.md

```diff
@@ -34,5 +34,16 @@ You should have access to an OpenShift cluster and be logged in with the
    $ curl http://`oc get routes/scala-spark-webapp --template='{{.spec.host}}'`
      Scala Akka SparkPi server running. Add the 'sparkpi' route to this URL to invoke the app.
    $ curl http://`oc get routes/scala-spark-webapp --template='{{.spec.host}}'`/sparkpi
-     Pi is roughly 3.1458957294786476  
+     Pi is roughly 3.1458957294786476
    ```
+
+ ### Optional parameter
+
+ If you would like to change the number of samples that are used to calculate
```

Comment:
this is fine, but I wonder if it might be better to call the parameter something other than `partitions` (e.g., `scale`).  The fact that the number of samples is a function of the number of partitions is an implementation detail of the code that probably shouldn't leak to the API (and might confuse users -- especially those new to Spark).

Reply:
i think `scale` makes good sense, and i agree it's clearer for a new user what that might mean. we'll need to adjust the python tutorial as well, but not a big deal.